applications_tests/gls_sharp_navier_stokes_3d/almost_blocked_channels.prm

```diff
@@ -100,6 +100,6 @@ subsection non-linear solver
 end
 
 subsection linear solver
-  set verbosity = quiet
-end
-
+  set verbosity         = quiet
+  set relative residual = 1e-1
```

Comment:
It was because the problem was too stiff, so instead I reduced the forcing term. With the current setup the solver decreases the residual by 6 orders of magnitude.